src/inc/switches.h

```diff
@@ -102,6 +102,10 @@
 #define OS_PAGE_SIZE PAGE_SIZE
 #endif
 
+#ifndef THREAD_MIN_STACK_SIZE
+#define THREAD_MIN_STACK_SIZE (64 * 1024) // 64 KB is a reasonably portable minimum between Windows and Unix
+#endif
```

Comment:
@jkotas mentions 128k in https://github.com/dotnet/coreclr/issues/9167 with regards to sufficient stack size on 64-bit... ?